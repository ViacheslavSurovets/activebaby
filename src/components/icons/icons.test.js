import React from 'react';
import { shallow } from 'enzyme';
import * as icon from '@icons';


describe ( 'icons', () => {
  it ( 'icons Snapshot', () => {
    expect ( shallow ( <icon.FailIcon /> ) ).toMatchSnapshot ();
    expect ( shallow ( <icon.CheckedIcon /> ) ).toMatchSnapshot ();
    expect ( shallow ( <icon.CloseIcon /> ) ).toMatchSnapshot ();
    expect ( shallow ( <icon.InstagramIcon /> ) ).toMatchSnapshot ();
    expect ( shallow ( <icon.LocationMarkerIcon /> ) ).toMatchSnapshot ();
    expect ( shallow ( <icon.LoginIcon /> ) ).toMatchSnapshot ();
    expect ( shallow ( <icon.SearchIcon /> ) ).toMatchSnapshot ();
    expect ( shallow ( <icon.ShopCartIcon /> ) ).toMatchSnapshot ();
    expect ( shallow ( <icon.VkIcon /> ) ).toMatchSnapshot ();
  } );
} );
