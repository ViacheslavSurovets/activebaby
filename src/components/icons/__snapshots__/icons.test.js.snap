// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons icons Snapshot 1`] = `ShallowWrapper {}`;

exports[`icons icons Snapshot 2`] = `ShallowWrapper {}`;

exports[`icons icons Snapshot 3`] = `ShallowWrapper {}`;

exports[`icons icons Snapshot 4`] = `ShallowWrapper {}`;

exports[`icons icons Snapshot 5`] = `ShallowWrapper {}`;

exports[`icons icons Snapshot 6`] = `ShallowWrapper {}`;

exports[`icons icons Snapshot 7`] = `ShallowWrapper {}`;

exports[`icons icons Snapshot 8`] = `ShallowWrapper {}`;

exports[`icons icons Snapshot 9`] = `ShallowWrapper {}`;
